### PR TITLE
MNT mark tests requiring internet

### LINKED ISSUE #43: Suggestion: Add marker for slow tests
Some tests take quite long to run. WDYT about adding a [marker for slow tests](https://docs.pytest.org/en/7.1.x/example/simple.html#control-skipping-of-tests-according-to-command-line-option)? Of course, in CI, we would always run the slow tests, but for local development, they will often be unnecessary.

### PR DESCRIPTION
**Pull request by adrinjalali**
Marking the test which requires internet, and is much slower than others.

Fixes #43 

cc @skops-dev/maintainers

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -11,3 +11,6 @@ filterwarnings = [
     "error::DeprecationWarning",
     "error::FutureWarning",
 ]
+markers = [
+    "network: marks tests as requiring internet (deselect with '-m \"not slow\"')",
```
Comment (BenjaminBossan) [inline pyproject.toml:15]:
```suggestion
    "network: marks tests as requiring internet (deselect with '-m \"not network\"')",
```

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -55,6 +55,21 @@ Using conda
 You can also replace the above `mamba` commands with `conda` if you don't have
 `mamba` installed.
 
+
+Running Tests
+~~~~~~~~~~~~~
+
+Certain tests require internet access to run, and they typically take slightly
```
Comment (BenjaminBossan) [inline CONTRIBUTING.rst:62]: "slightly" :D

**Comment by adrinjalali** [comment]
Love a green CI!

**Comment by BenjaminBossan** [comment]
@adrinjalali I resolved a merge conflict. If you're fine with the outcome, let me know and we can merge this.

**Comment by adrinjalali** [comment]
Yes. looks good to me!

**Review by BenjaminBossan — APPROVED** [review]
Thanks, useful addition.

**Pull request merged**